src/Components.tsx

```diff
@@ -59,7 +59,7 @@ export const ConnectAccountOnboarding = ({
   );
 
   useAttachAttribute(onboarding, 'privacy-policy-url', privacyPolicyUrl);
```

Comment:
You'll have to bump the version in `devDependency` and `peerDependency` of the @stripe/connect-js package once published

Reply:
Done!